### DISCUSSION TITLE
Add other solutions to the Django doc

### DISCUSSION THREAD

**Discussion by michael-yin** [themesberg/flowbite#192] [General]
The current Django doc https://flowbite.com/docs/getting-started/django/ is great to beginner user who wants to integrate Tailwind CSS.

It would be great if it has a `Other Solutions` section which contains some different approach.

For example, https://python-webpack-boilerplate.readthedocs.io/en/latest/setup_with_tailwind/

With the above solution, developer can also quickly make `flowbite` work with `Flask`, which is also popular in Python community.

Thx.

**Comment by zoltanszogyenyi** [comment]
Hey @michael-yin,

If you'd like to you can open up a PR or just write an integration guide for Flask similar to our Django guide and I will post it on the documentation :)

Cheers,
Zoltan

PS: also moved this to the discussions area

**Comment by michael-yin** [comment]
@zoltanszogyenyi 

Hi, I already created PR https://github.com/themesberg/flowbite/pull/217

The guide teach people to use `python-webpack-boilerplate` to create frontend project which bundled by Webpack, and then install Tailwind CSS and `flowbite`

If you have any question, please let me know.

Thx.

**Comment by zoltanszogyenyi** [comment]
I have checked it out and thanks for the contribution.

Would it be possible if rather than changing the current setup (which is good for beginners as you've stated) and add the Webpack configuration as a secondary solution? (ie. create another heading after the main setup, something like "Using Webpack").

Thanks!
